dags/atd_service_bot_index_issues_to_dts_portal.py

```diff
@@ -0,0 +1,70 @@
+import os
+from datetime import datetime, timedelta
+
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from onepasswordconnectsdk.client import Client, new_client
+import onepasswordconnectsdk
```

Comment:
I think this is wise since it has bit us before elsewhere. @frankhereford - I would update here, but it might be good to make the change into the `airflow-v2` branch since this PR will probably take longer to merge since we have a lot of good discussion to consider. What do you think?

Reply:
Yes, I am totally on board with pinning the requirements to a major version. I had intended to do that at some point, and just had let it slip though the cracks. 

I'll be happy to open a micro PR that does that pinning, and I'll get that done and up on the boards here momentarily.

Also yes - pendulum is something installed by the Airflow team in the base image on which we build our lightly customized image. It's always going to be available and we don't need to worry about requiring it.

Reply:
https://github.com/cityofaustin/atd-airflow/pull/104